server/api/products.spec.js

```diff
@@ -4,7 +4,7 @@ const { expect } = require('chai')
 const request = require('supertest')
 const db = require('../db')
 const app = require('../index')
-const Product = db.model('product')
+const Product = db.model('products')
```

Comment:
const {Product} = require('../db/models') works with the api route on my machine.  Please verify.